regression/esbmc-cpp/cpp/github_2389_1/test.desc

```diff
@@ -0,0 +1,4 @@
+CORE
+main.cpp
+--class "A B" --function test
+^VERIFICATION SUCCESSFUL$
```

Comment:
Shouldn't it give verification failed? There is an assert(0) in the test method of class B.

Reply:
Only the first matched one will be verified